README.md

```diff
@@ -50,12 +50,9 @@ Now include the javascript file in each of your iframes in your fixtures or test
 Now insert calls at each point in your tests where a new piece of UI becomes visible or exposed:
 
 ```js
-axe.run(function (err, results) {
-  if (err) throw err;
-    ok(results.violations.length === 0, 'Should be no accessibility issues');
-    // complete the async call
-    ...
-});
+axe.run().then(results => {
+  assert.isEmpty(results.violations, 'Has no accessibility issues');
+}).catch(done);
```

Comment:
What is `done`?

Reply:
when using it in a test, you often get a `done` function to call. Open to alternative suggestions. Any thoughts?

Reply:
IMO *ALL* code snippets in documentation should be copy/paste/run-able. What about this instead:

```js
axe.run()
  .then(results => {
    if (results.violations.length) {
      throw new Error('Accessibility issues found')
    }
  })
  .catch(err => {
    console.error('Something bad happened:', err.message)
  })
```